ui/packages/catalog-ui-search/src/main/webapp/component/builder/builder.view.js

```diff
@@ -71,6 +71,9 @@ module.exports = Marionette.LayoutView.extend({
     } else {
       this.model.set(AVAILABLE_TYPES_ATTRIBUTE, retrievedAvailableTypes)
     }
+
+    this.ingestUrl =
+      this.options.ingestUrl? this.options.ingestUrl : './internal/catalog/?transform=geojson'
```

Comment:
:pencil2: Since this module is already being copied downstream, we could probably forego the changes here and just move them all at once when we contribute them back into DDF.